igvc/src/pathplanner/main.cpp

```diff
@@ -115,8 +115,7 @@ int main(int argc, char** argv)
 
   if(!pNh.hasParam("goal_threshold") || !pNh.hasParam("threshold") || !pNh.hasParam("speed") || !pNh.hasParam("baseline") || !pNh.hasParam("minimum_omega") || !pNh.hasParam("maximum_omega") || !pNh.hasParam("delta_omega") || !pNh.hasParam("point_turns_enabled") || !pNh.hasParam("reverse_enabled") || !pNh.hasParam("max_obstacle_delta_t") || !pNh.hasParam("alpha") || !pNh.hasParam("beta") || !pNh.hasParam("bounding_distance")) {
     ROS_ERROR_STREAM("path planner does not have all required parameters");
-    ROS_ERROR_STREAM("path planner does not have all required parameters");
-    ROS_ERROR_STREAM("path planner does not have all required parameters");
+    ros::shutdown();
```

Comment:
You should just return from main() here, rather than calling shutdown(). ros::shutdown(), in fact, does not actually terminate the program. It's responsible for cleaning up a lot of the behind-the-scenes resources in ROS, but this functionality is taken care of when the NodeHandle is destructed.
From the ROS documentation:
"It is generally not necessary to call this function, as the node will automatically shutdown when all NodeHandles destruct. However, if you want to break out of a spin() loop explicitly, this function allows that." [[ref](http://docs.ros.org/api/roscpp/html/namespaceros.html#a2d29113188d411e78f2c221aa7a7473e)]